gradle-plugin-code/build.gradle

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'java-gradle-plugin'
+    id 'com.gradle.plugin-publish' version '0.10.1'
+}
+
+group 'com.trivago.rta'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    jcenter()
+    mavenLocal()
+    maven {
+        url = 'http://repo.maven.apache.org/maven2'
+    }
+}
+
+project.description     = 'Cluecumber Gradle Plugin for Cucumber Reports'
+project.group           = 'com.trivago.rta'
+project.version         = '2.3.2'
+
+project.ext.name        = 'cluecumber-report-gradle-plugin'
+project.ext.scm         = 'https://github.com:trivago/cluecumber-report-plugin.git'
+project.ext.url         = 'git@github.com:trivago/cluecumber-report-plugin.git'
+
+dependencies {
```

Comment:
Why is it necessary to have all dependencies here again? Wouldn't it be better to have them all in the core and just use core here?